modules/govuk_ci/manifests/agent.pp

```diff
@@ -54,4 +54,10 @@
   package { 'libgdal-dev': # needed for mapit
     ensure => installed,
   }
+
+  cron::crondotdee { 'clean_up_bundles':
+    command => 'find /var/lib/jenkins/bundles/* -maxdepth 0 -type d -mtime +1 -exec rm -rf {} \;',
```

Comment:
I think this expression needs to change:
```
lauramartin@integration-ci-agent-1:~$ sudo find /var/lib/jenkins/bundles/* -maxdepth 0 -type d -mtime +1
find: `/var/lib/jenkins/bundles/*': No such file or directory
```

I also think it would be wise to not remove the root path of the search, so do something like:
```
lauramartin@integration-ci-agent-1:~$ sudo find /var/lib/jenkins/bundles/ -mindepth 2 -maxdepth 2 -type d -mtime +1 |head
/var/lib/jenkins/bundles/govuk-dns-config/update-dns-zone
/var/lib/jenkins/bundles/govuk-dns-config/add-new-bad-record-test
/var/lib/jenkins/bundles/govuk-dns-config/validate_dns
/var/lib/jenkins/bundles/govuk-dns-config/remove-panopticon
/var/lib/jenkins/bundles/govuk-dns-config/remove-backslash-dmarc
/var/lib/jenkins/bundles/govuk-dns-config/master
/var/lib/jenkins/bundles/specialist-frontend/test-push
/var/lib/jenkins/bundles/specialist-frontend/deployed-to-production
/var/lib/jenkins/bundles/government-frontend/news-articles
/var/lib/jenkins/bundles/government-frontend/fix-fatality-notice-preview
```